dev-docs/RFCs/v7.0/framebuffer-texture-api-rfc.md

```diff
@@ -24,23 +24,26 @@ The problem is that:
 
 ## Proposals
 
-The key proposal is to move methods from Framebuffer and Texture into global functions.
+* The key proposal is to move methods from Framebuffer and Texture into global functions.
 
 > TBD - some work remains to unify the proposal
 
 | Method                         | Replacement |
 | ---                            | ---         |
-| `Frambuffer.copyToDataUrl`     |  copyFramebufferToDataUrl |
-| `Frambuffer.copyToImage`       |  copyFramebufferToImage |
-| `Frambuffer.copyToFramebuffer` |  copyFramebuffer |
-| `Frambuffer.copyToTexture`     |  copyFramebufferToTexture |
-| `Frambuffer.blit`              |  blitFramebuffer |
+| `Framebuffer.readPixels`       |  `copyToArray` |
+| `Framebuffer.readPixelsToBuffer`|  `copyToBuffer` |
+| `Frambuffer.copyToDataUrl`     |  `copyToDataUrl` |
+| `Frambuffer.copyToImage`       |  `copyToImage` |
+| `Frambuffer.copyToFramebuffer` |  `copyFramebuffer` |
+| `Frambuffer.copyToTexture`     |  `copyToTexture` |
+| `Frambuffer.blit`              |  `blit` |
+| `Texture.copyFramebuffer`      |  `copyToTexture` |
 | `Texture.setImageData`         |  setTextureImageData |
 | `Texture.setSubImageData`      |  setTextureSubImageData |
 | `Texture.setImage3D`           |  setImage3D |
 
-
+* Make all methods that accept `Framebuffer` objects as source or target, to accept `Texture` object in addition to `Framebuffer` object. Internally we will implement a utility method to wrap `Texture` object into a `Framebuffer` object.
 
 ## Deprecation
 
-All removed methods will be stubbed and emit a deprecation message pointing to the luma.gl upgrade guide.
\ No newline at end of file
+All removed methods will be stubbed and emit a deprecation message pointing to the luma.gl upgrade guide.
```

Comment:
Note: If we just deprecate these we get none of the tree-shaking benefits. Given that this is a major release, and these are not the most frequently used functions, I would propose that we do not give a "deprecation warning" but rather an "error please upgrade" message.